Atomic/atomic.py

```diff
@@ -30,6 +41,22 @@
 from docker.errors import NotFound
 
 IMAGES = []
+ATOMIC_LIBEXEC = os.environ.get('ATOMIC_LIBEXEC', '/usr/libexec/atomic')
+
+OSTREE_DOCKERIMG_PREFIX = "dockerimg/"
+SYSTEMD_UNIT_FILES_DEST = "/etc/systemd/system"
```

Comment:
thinking more of it, it should not make a difference if we call it in another way, e.g. `ociimage/`, being a Docker layer is just a detail of how we will handle it, so I am going to change it